packaging/suse/portus.spec.in

```diff
@@ -28,7 +28,13 @@ Release:        0.0.1
 License:        Apache-2.0
 Summary:        Authorization service and fronted for Docker registry (v2)
 Url:            https://github.com/SUSE/Portus
-Source:         %{branch}.tar.gz
+Source0:        %{branch}.tar.gz
+
+# Generated with `yarn install` which produces a reproduceable `node_modules`
+# directory thanks to the yarn.lock file defined in the Portus repo.
+Source1:        node_modules.tar.gz
+
```

Comment:
> could we create this in travis and upload it to the open build service?

Yes, that's something that I intent to do as a follow-up of this PR (so, in another PR 😉).

> could you add a bit more explanation on how to generate the node_modules?

I'll add it on the README fil inside of `packaging/suse`.

> May the node_modules be part of the yarn rpm?

No, because in the yarn rpm there should only be the node_modules of yarn. The way I think about this is that the `node_modules` directory is like the `vendor` directory in Go programs, but instead of having them inside of the code, we rely on a yarn.lock in order to generate it. It's kind of a mix between the Go vendor solution and the bundler solution if that makes sense 😅